clouddriver-security/src/main/groovy/com/netflix/spinnaker/clouddriver/security/Permissions.groovy

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.security
+
+import com.netflix.spinnaker.fiat.model.Authorization
+import groovy.transform.InheritConstructors
+
+/**
+ * A Permissions object is a map of Authorizations to groups/roles. A user generally needs only be
+ * a member of at least one group in the list to be authorized for that permission.
+ */
+@InheritConstructors
+class Permissions extends HashMap<Authorization, List<String>> {
```

Comment:
Everything else in the account credentials is immutable - can you make this immutable as well